data/json/monsterdrops/zombie_survivor.json

```diff
@@ -87,5 +139,30 @@
       { "group": "survivor_melee", "prob": 40 },
       { "group": "guns_survival", "prob": 60 }
     ]
+  },
+  {
+    "id": "survivor_longguns",
+    "type": "item_group",
+    "subtype": "distribution",
+    "entries": [
+      {
+        "distribution": [
+          { "group": "guns_smg_improvised", "prob": 10 },
+          { "group": "guns_rifle_improvised", "prob": 100 },
+          { "group": "guns_shotgun_improvised", "prob": 100 },
+          { "group": "guns_launcher_improvised", "prob": 30 }
+        ],
+        "prob": 200
+      },
+      { "item": "ar15", "prob": 5, "charges": [ 0, 30 ] },
+      { "item": "mossberg_500", "prob": 5, "charges": [ 0, 8 ] },
+      { "item": "remington_870", "prob": 5, "charges": [ 0, 6 ] },
+      { "item": "ruger_1022", "prob": 15, "charges": [ 0, 10 ] },
+      { "item": "shotgun_410", "prob": 10, "charges": [ 0, 1 ] },
+      { "item": "shotgun_s", "prob": 10 },
+      { "item": "shotgun_d", "prob": 10, "charges": [ 0, 2 ] },
+      { "item": "sten", "prob": 20, "charges": [ 0, 32 ] },
+      { "item": "sks", "prob": 10, "charges": [ 0, 10 ] }
```

Comment:
Are the probabiblites supposed to add up to 90 rather than 100?

Reply:
Probabilities don't sum up here. They just mean that, e.g. `ar15` has four times less chance to spawn than `sten` has.